frontend/packages/knative-plugin/src/components/overview/FilterTable.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+import cx from 'classnames';
+import { TableVariant, Table, TableBody } from '@patternfly/react-table';
+import './FilterTable.scss';
+
+export type FilterTableRowProps = { key: string; value: string }[];
+type FilterTableProps = {
+  filters: FilterTableRowProps;
+  bordered?: boolean;
+};
+
+const FilterTable: React.FC<FilterTableProps> = ({ filters, bordered = true }) => {
+  const filterRow = (key: string, value: string) => {
+    const className = cx({ 'table-row-bordered': bordered });
+    return {
+      cells: [
+        {
+          title: key,
+          props: {
+            colspan: 3,
```

Comment:
Did you mean `colSpan`? Getting this warning.
<img width="832" alt="Screenshot 2020-07-29 at 12 00 35 PM" src="https://user-images.githubusercontent.com/2561818/88764734-1e7b0d80-d193-11ea-84ac-0d3121d7cd72.png">

Reply:
Fixed colSpan issue, as it was throwing this error if we provide the colSpan in both the cells